packages/@react-stately/color/src/useHexColorFieldState.ts

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {Color} from './Color';
+import {ColorInput, HexColorFieldProps} from '@react-types/color';
+import {NumberFieldState} from '@react-stately/numberfield';
+import {useCallback, useState} from 'react';
+import {useControlledState} from '@react-stately/utils';
+
+export interface HexColorFieldState extends NumberFieldState {
```

Comment:
```suggestion
export interface HexColorFieldState extends Omit<NumberFieldState, 'value' | 'setValue'> {
```

Reply:
this should allow you remove them from your return object and should be clearer on what the type is

Reply:
Resolved. Used `Omit` to remove `value` and `setValue` from the return object